models/pull.go

```diff
@@ -854,6 +854,45 @@ func (pr *PullRequest) PushToBaseRepo() (err error) {
 	return nil
 }
 
+// SetTargetBranch sets the target branch of this pull request
+func (pr *PullRequest) SetTargetBranch(targetBranch string, doer *User) (err error) {
+	oldBranch := pr.BaseBranch
+	pr.BaseBranch = targetBranch
+
+	sess := x.NewSession()
+	defer sess.Close()
+
+	// Regenerate patch and test conflict.
+	if err = pr.UpdatePatch(); err != nil {
```

Comment:
This does not seem to work yet. :(

Reply:
Merge of #9302 would not require this change (or I had to remove it), right? Might not be an issue then.

Reply:
Maybe you can wait that PR.